react-sharing-state/my-app/src/Container.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+
+/**
+ * A container of items.
+ * One item is displayed at a time, with buttons to flip through them:
+ * - Next and Prev scroll forward and backward one item
+ * - An array of buttons scroll to a specific item
+ * TODO: The buttons don't work correctly!
+ */
+export default function Container({ items }) {
+  const [current, setCurrent] = useState(0);
+  const [backgroundColor, setBackgroundColor] = useState('white');
+
+  return (
+    <div>
+      <div>{items[current]}</div>
+      <div>
+        <Button text="Prev"
+          onClick={() => setCurrent(((current - 1) + items.length) % items.length)}
+          style={() => setBackgroundColor(backgroundColor)}
+          />
+        <Indicator count={items.length} current={current} setCurrent={setCurrent}/>
+        <Button text="Next"
+          onClick={() => setCurrent((current + 1) % items.length)}
+          />
+      </div>
+    </div>
+  );
+}
+
+/**
+ * A button that toggles its color between white and lightblue.
+ * TODO: Remove the toggle behavior and make the background color a prop, default white.
+ * TODO: When clicked, change the current item in the Container.
+ */
+function Button({ text, onClick, backgroundColor = 'white' }) {
+  return <button onClick={onClick}
+                 style={{ backgroundColor }}
+          >
+          {text}
+          </button>;
+  }
+
+/**
+ * An array of buttons.
+ * TODO: Highlight only the active button lightblue.
+ */
+function Indicator({ count, current, setCurrent, backgroundColor }) {
```

Comment:
According to convention, the `setCurrent` prop should be named starting with `on`, so perhaps `onSetCurrent`?

Reply:
Changed `setCurrent` to `onSetCurrent`

Reply:
<img width="853" alt="Screenshot 2023-03-31 at 2 09 13 PM" src="https://user-images.githubusercontent.com/110208589/229231992-c7f50a5b-3ae7-40ac-91be-f85f98111d41.png">